protocol/xmrmaker/backend_offers.go

```diff
@@ -24,12 +24,21 @@ func (b *Instance) MakeOffer(
 		return nil, errUnlockedBalanceTooLow{unlockedBalance, o.MaxAmount}
 	}
 
-	extra, err := b.offerManager.AddOffer(o, relayerEndpoint, relayerFee)
+	if relayerFee != nil {
+		if o.EthAsset != types.EthAssetETH {
+			return nil, errRelayingWithNonEthAsset
+		}
+		if relayerFee.Cmp(relayer.MinRelayerFeeEth) < 0 {
+			return nil, errRelayerFeeTooLow{relayer.MinRelayerFeeEth, relayerFee}
+		}
```

Comment:
if we're going to force the relayer fee to be fixed, we should probably just remove the ability for someone to set the fee they want to pay.

Reply:
In the name of simplicity, at least in the short term, I was thinking the same thing.  I can make that change. I wasn't ruling out the possibility of providing an API to set minimum relayer fees in the future. We'd need to design any complexity in carefully though. In the short term, we need to maximize the change that Bob can claim. Any design needs to include how we recover when Bob can't claim immediately.